lms/static/scripts/frontend_apps/utils/api.js

```diff
@@ -47,16 +47,18 @@ export class ApiError extends Error {
  */
 async function apiCall({ path, authToken, data }) {
   let body;
+  const headers = {
+    Authorization: authToken,
+  };
   if (data !== undefined) {
     body = JSON.stringify(data);
+    headers['Content-Type'] = 'application/json; charset=UTF-8';
   }
 
   const result = await fetch(path, {
     method: data === undefined ? 'GET' : 'POST',
     body,
-    headers: {
-      Authorization: authToken,
-    },
+    headers: headers,
```

Comment:
```suggestion
    headers,
```

You can shorten eg. ` { propertyA: propertyA, propertyB: propertyB }` to `{ propertyA, propertyB }` and we usually do.

Reply:
Cool. I've added that. I'll wait for the tests to pass and merge I think.